File: SDKs/clubhouseapi.com/1/python/src/sdk/models/operations/post_follow.py
from dataclasses import dataclass, field
from typing import Any,Optional


@dataclass
class PostFollowRequest:
    request: Optional[dict[str, Any]] = field(default=None, metadata={'request': { 'media_type': 'application/json' }})
    

@dataclass
class PostFollowResponse:
    content_type: str = field(default=None)
    status_code: int = field(default=None)
    

File: SDKs/clubhouseapi.com/1/typescript/src/sdk/models/operations/postfollow.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";



export class PostFollowRequest extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "request, media_type=application/json" })
  request?: Map<string, any>;
}


export class PostFollowResponse extends SpeakeasyBase {
  @SpeakeasyMetadata()
  contentType: string;

  @SpeakeasyMetadata()
  statusCode: number;
}
